VxWorks/Os/Task.cpp

```diff
@@ -1,11 +1,14 @@
-#include <Os/Task.hpp>
-#include <Fw/Types/Assert.hpp>
-
 #include <vxWorks.h>
 #include <taskLib.h> // need it for VX_FP_TASK
 #include <sysLib.h>
 #include <tickLib.h>
 
+
+#define VX_OK 0
```

Comment:
This is okay for now. It'll be better to have this define in a higher level file such that our other VxWorks/Os/*.cpp files have access to this definition.